src/mavlink/mavlink_camera.rs

```diff
@@ -373,15 +373,41 @@ fn receive_message_loop(
                             error!("Failed to parse control id: {:#?}", error);
                             continue;
                         }
-                        let _control_id = control_id.unwrap();
+                        let control_id = control_id.unwrap();
 
                         if let Err(error) = control_value {
                             error!("Failed to parse parameter value: {:#?}", error);
                             continue;
                         }
-                        let _control_value = control_value.unwrap();
+                        let control_value = control_value.unwrap();
 
-                        //TODO: Control V4L
+                        let information = mavlink_camera_information.as_ref().lock().unwrap();
+                        let mut param_result = mavlink::common::ParamAck::PARAM_ACK_ACCEPTED;
+                        if let Err(error) = information
+                            .video_source_type
+                            .inner()
+                            .set_control_by_id(control_id, control_value)
+                        {
+                            error!(
+                                "Failed to set parameter {:?} with value {:?}. Reason: {:#?}",
+                                control_id, control_value, error
+                            );
+                            param_result = mavlink::common::ParamAck::PARAM_ACK_FAILED;
+                        }
```

Comment:
To avoid missing the drop, you can avoid creating an intermediary component since information is only used once in this scope. 
```suggestion
                        let mut param_result = mavlink::common::ParamAck::PARAM_ACK_ACCEPTED;
                        if let Err(error) = mavlink_camera_information
                            .as_ref()
                            .lock()
                            .unwrap()
                            .video_source_type
                            .inner()
                            .set_control_by_id(control_id, control_value)
                        {
                            error!(
                                "Failed to set parameter {:?} with value {:?}. Reason: {:#?}",
                                control_id, control_value, error
                            );
                            param_result = mavlink::common::ParamAck::PARAM_ACK_FAILED;
                        }
```